### ISSUE TITLE
Contours: does not work well with multiple symmetry enabled

### ISSUE THREAD

**Issue by vxlcoder** [CGCookie/retopoflow#397]
if cutting with symmetry along multiple axes, the counts are not correct and the verts don't go all the way to the symmetry plane.
![image](https://user-images.githubusercontent.com/3790699/34321748-91388078-e7e4-11e7-84b6-05d181a0da34.png)

**Comment by vxlcoder** [comment]
i believe this issue is resolved!

**Comment by carter2422** [comment]
I'm afraid this is still very problematic. This is what I get on a default cube when mirroring both X and Y: https://cl.ly/tFha

**Comment by vxlcoder** [comment]
will take another look

**Comment by jfranmatheu** [comment]
This one seems to be ok when mirroring in multiple axis over the default cube as long as you do the contours parallel to the planes. Should it work when tracing in perpendicular direction as well?

https://github.com/user-attachments/assets/0add29d3-4d4d-4d3a-aba9-ac75ed5acc3f

**Comment by jlampel** [comment]
Yeah, I think the issue is when going in the other direction. I don't think we need to worry about this for v3, but I think it would be good to fix in v4 at some point. 

https://github.com/user-attachments/assets/12b904ef-9cfd-4e6c-aa50-539142334690

**Comment by jlampel** [comment]
Fixed! Each axis was expecting a cyclic loop, so when it was already clipped the second axis got a bad input. Now the first axis clips the loop and any second or third axes clip the segment.
